mysite/_config/theme.yml

```diff
@@ -6,3 +6,10 @@ SilverStripe\View\SSViewer:
     - '/mysite'
     - 'simple'
     - '$default'
+
+SilverStripe\LoginForms\EnablerExtension:
+  login_themes:
+    - '$public'
+    - 'silverstripe/login-forms:login-forms'
+    - 'simple'
```

Comment:
I was getting some errors because login form couldn't find a bunch of style sheet.

Reply:
Likely caused by [this issue](https://github.com/silverstripe/silverstripe-login-forms/issues/44).